tests/test_worker.py

```diff
@@ -58,6 +58,25 @@ def test_worker_stop(hq_env: HqEnv):
     wait_for_worker_state(hq_env, 1, "STOPPED")
     hq_env.check_process_exited(process)
 
+    response = hq_env.command(["worker", "stop", "1"])
+    assert "worker is already stopped" in response
+    response = hq_env.command(["worker", "stop", "2"])
+    assert "worker not found" in response
+
+
+def test_worker_stop_all(hq_env: HqEnv):
+    hq_env.start_server()
+    process_w_1, process_w_2 = hq_env.start_worker(), hq_env.start_worker()
+
+    wait_for_worker_state(hq_env, 1, "RUNNING")
+    wait_for_worker_state(hq_env, 2, "RUNNING")
+    hq_env.command(["worker", "stop", "all"])
+    wait_for_worker_state(hq_env, 1, "STOPPED")
```

Comment:
You can also use `wait_for_worker_state(hq_env, [1, 2], "STOPPED")` to wait for multiple workers.